pkgs/development/libraries/ogre/default.nix

```diff
@@ -30,6 +39,7 @@ stdenv.mkDerivation {
      libX11 libXmu libSM pkgconfig
      libXxf86vm libICE
      libXrender
+     unzip
```

Comment:
This is a buildNative input — which probably also explains the problem with unpack hook not working.

Reply:
Updated as per this suggestion -- worked great, thanks!